use/posts.js

```diff
@@ -127,10 +129,8 @@ export const usePosts = () => {
       hideLabel: true,
       type: 'textarea',
       placeholder: 'Text (markdown)',
```

Comment:
that should be also translated

Reply:
changes made please review

Reply:
it's enough just to click "Resolve conversation" ;)